app/assets/stylesheets/components/_moving-guide.scss

```diff
@@ -0,0 +1,66 @@
+.moving-guide {
+  .location-section {
+    padding-top: 2rem;
+
+    img {
+      width: 3em;
```

Comment:
I actually chose `em` specifically because I have a feeling that there may be a debate in the future about the font size, given the difference between the Zeplin designs and the current implementation. If the font size is changed later, then this image will still be roughly the same size relative to the text next to it.